qiskit/providers/ibmq/api_v2/session.py

```diff
@@ -138,6 +138,13 @@ def request(self, method, url, bare=False, **kwargs):
             # Wrap the requests exceptions into a IBM Q custom one, for
             # compatibility.
             message = str(ex)
+            if ex.response is not None:
+                try:
+                    message += ". {}".format(ex.response.json()['error']['message'])
```

Comment:
Can you try to include also `code` in the message? Even if `message` is sufficient for the user, it would be nice to also have the code to uniquely identify the error for debugging/support purposes.